.github/actions/check_files/rpm5_linux_agent_amd64.csv

```diff
@@ -1,58 +1,58 @@
 full_filename,owner_name,group_name,mode,type,prot_permissions,size_bytes,size_error
-/var/ossec/tmp/sca-4.*.el5-tmp/rocky/cis_rocky_linux_9.yml,root,wazuh,640,file,-rw-r-----,300524,0.1
```

Comment:
This changes the behaviour, its really expected?
![image](https://github.com/user-attachments/assets/1d6e3d96-ab26-4c96-923a-4f6ff56dc199)
![image](https://github.com/user-attachments/assets/34eadbdb-dc95-48c2-a04e-5ad5f9ebcaab)

Reply:
Yes, since the **version** always comes with a **_revision_**. 
So it will always come `4.<minor>.<patch>-<revision>`.

Reply:
Perfect, then its useful to prevent non expected file names